cmd/kueuectl/app/list/list_pods.go

```diff
@@ -228,6 +228,9 @@ func (o *PodOptions) getPodLabelSelector() (string, error) {
 		return "", nil
 	}
 
+	if cbs.NewJob == nil {
+		return "", fmt.Errorf("NewJob is not defined for integration job")
```

Comment:
```suggestion
		return "", nil
```

We already have an error [here](https://github.com/kubernetes-sigs/kueue/blob/c7b367ec851b862f831fa5df955598702f0116ee/cmd/kueuectl/app/list/list_pods.go#L107).